io.c

```diff
@@ -4942,8 +4943,20 @@ rb_io_readchar(VALUE io)
  *    f.getbyte # => 209
  *    f.close
  *
- *  Related: IO#readbyte (may raise EOFError).
+ *    File.read('t.dat')
+ *    # => "\xFE\xFF\x99\x90\x99\x91\x99\x92\x99\x93\x99\x94"
+ *    File.read('t.dat')
+ *    # => "\xFE\xFF\x99\x90\x99\x91\x99\x92\x99\x93\x99\x94"
+ *    f = File.new('t.dat')
+ *    f.getbyte # => 254
+ *    f.getbyte # => 255
+ *    f.seek(-2, :END)
+ *    f.getbyte # => 153
+ *    f.getbyte # => 148
+ *    f.getbyte # => nil
+ *    f.close
```

Comment:
What's the purpose of adding these examples? Adding this makes the examples very long, could we shorten it with the example above?

Reply:
Removed.